### DISCUSSION TITLE
Site moves when opening drawer

### DISCUSSION THREAD

**Discussion by ghost** [saadeghi/daisyui#1533] [Q&A] [answered]
Can we disable that tiny movement of site's content happening when you open drawer?

I've used this class from [reply](https://github.com/saadeghi/daisyui/issues/1067#issuecomment-1229579794) 

but it only disables drawers animation

**Comment by DrewWalsh** [comment, marked as answer]
I was able to fix this by adding this to my unscoped scss. 

```
.drawer-toggle:checked ~ .drawer-content {
	--tw-translate-x: 0rem !important;
}
```

By default, this gets set to 0.5rem when the drawer opens, shifting the whole page to the right a little bit, no idea why.
